examples/provider_store/README.md

```
@@ -51,6 +56,14 @@ $ pc supply order 123456
 ```
 
 
+## Parts
+
+### nut (alias to hardware/nut_m4_0_7mm)
```

Markdownlint (0.37.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)